packages/eslint-plugin-specs/react-native-modules.js

```diff
@@ -36,6 +37,7 @@ function requireModuleParser() {
       withBabelRegister(config, () => {
         RNModuleParser = require('@react-native/codegen/src/parsers/flow/modules');
         RNParserUtils = require('@react-native/codegen/src/parsers/utils');
+        RNFlowParser = require('@react-native/codegen//src/parsers/flow/parser');
```

Comment:
```suggestion
        RNFlowParser = require('@react-native/codegen/src/parsers/flow/parser');
```

Reply:
Ups my bad, I rebased too quickly. Thanks for the catch!